From 8c4619f67ddbc27be79fb5381322f349eff81d7a Mon Sep 17 00:00:00 2001
From: Tobias <ruck.tobias@gmail.com>
Date: Sun, 20 Jun 2021 15:08:38 -0500
Subject: [PATCH 123/196] Reset default minimumchainwork and assumevalid for
 mainnet. (#205)

This will get nodes out of IBD immediately and allow mining without changing configs.
---
 src/networks/abc/chainparamsconstants.cpp | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/networks/abc/chainparamsconstants.cpp b/src/networks/abc/chainparamsconstants.cpp
index efe864123..606d9795f 100644
--- a/src/networks/abc/chainparamsconstants.cpp
+++ b/src/networks/abc/chainparamsconstants.cpp
@@ -5,8 +5,8 @@
 #include <chainparamsconstants.h>
 
 namespace ChainParamsConstants {
-    const BlockHash MAINNET_DEFAULT_ASSUME_VALID = BlockHash::fromHex("00000000000000005ce0908e5a27e2f4c202f9f44445afff0899d254439f8480");
-    const uint256 MAINNET_MINIMUM_CHAIN_WORK = uint256S("00000000000000000000000000000000000000000153f1c5a543971b7f71bff5");
+    const BlockHash MAINNET_DEFAULT_ASSUME_VALID = BlockHash::fromHex("0000000000000000000000000000000000000000000000000000000000000000");
+    const uint256 MAINNET_MINIMUM_CHAIN_WORK = uint256S("0000000000000000000000000000000000000000000000000000000000000000");
     const uint64_t MAINNET_ASSUMED_BLOCKCHAIN_SIZE = 210;
     const uint64_t MAINNET_ASSUMED_CHAINSTATE_SIZE = 3;
 
-- 
2.39.5 (Apple Git-154)

